asapdiscovery-alchemy/asapdiscovery/alchemy/tests/conftest.py

```diff
@@ -27,3 +28,16 @@ def tyk2_fec_network():
     """Create a FEC planned network from file"""
     fec_network = fetch_test_file("tyk2_small_network.json")
     return FreeEnergyCalculationNetwork.from_file(fec_network)
+
+
+@pytest.fixture()
```

Comment:
Haha looks like we had the same thought. Did something similar in #475; can sort out the dead once we merge them both.

Reply:
Yeah, I did notice that as well haha. I prefer yours since it's making some assertions that seem useful to have.